Bottom: 09ed451c6b5c30d2dbe0bbedc3379ef8034dc899
Top:    1c60f4db6798ea721b5d8e64b7989ffbd653b24b
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-04-29 20:11:35 -0700

librdmacm: Make ucma_init_all static

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/cma.c b/src/cma.c
index 0dc229e..4fa41ff 100644
--- a/src/cma.c
+++ b/src/cma.c
@@ -321,7 +321,7 @@ err:
 	return ret;
 }
 
-int ucma_init_all(void)
+static int ucma_init_all(void)
 {
 	int i, ret = 0;
